todo-frontend-angular/src/app/todolist-selector/todo.service.ts

```diff
@@ -19,7 +19,17 @@ export class TodoService {
   getTodos(id: string) {
     return this.http.get<Todolist>(`${this.todolistUrl}/${id}`).pipe(map(todolist => todolist.todos));
   }
-
+  addTodo(newTodo: NewTodoDTO){
+    return this.http.post<Todo>(`${this.todoUrl}`, newTodo,{
+      headers: new HttpHeaders({
+        'Content-Type': 'application/json',
+        'accept': '*/*'
+      })
+    });
+  }
+  getTodoLists(){
+    return this.http.get<Todolist[]>(`${this.todolistUrl}`).pipe(map(todoList=>todoList));
```

Comment:
I can see that you have added the baseUrl in environment :) Shouldn't you use that here?

Reply:
Have done that now